3.12.x

alexarobu opened the pull request:
Updated small blocks of text and various screen grabs to reflect changes happening in v3.12.x of GeoNetwork in comparison to v3.10.x

alexarobu committed:
update "Filter and sort options" section

Minor corrections to wording

alexarobu committed:
Changed "Print the map" section

Removed "JPEG" from the available print formats

alexarobu committed:
Update "Duplicating existing metadata" section

A slight change in the order of the steps - to include the need to click on an additional button. (button to be added later!)

alexarobu committed:
Update "Editing metadata" section

Changed the order of the steps in "Assigning a licence category", "Publishing metadata" and "Deleting metadata" sections to reflect the need to click an additional button in order to access the Contributor board. (button to be added later!)

alexarobu committed:
Update "Advanced search" section

Further explained the use of advanced search capabilities and added a couple of button icons to enhance the text

alexarobu committed:
Update "Advanced workflow" section

Changes to wording to reflect minor changes in items' position on screen

alexarobu committed:
Update "Harvester Settings" section

Wording updated to match what the portal shows in this section

alexarobu committed:
Updated "Directories" section

Updated the wording to reflect the changes in button naming and order and included instructions on how to reach the sub-section from the header menu.

alexarobu committed:
Update a few sub-sections

Minor changes to wording in the following sub-sections: "Manage users", "Settings", "INdex admin" and "Batch process"

alexarobu committed:
Update sub-section title

"Schematron" changes title to "Validation"

alexarobu committed:
Update "Catalogue admin tools" sub-section

Added an extra button and description of what it does to the existing list

alexarobu committed:
Added images to repo

Replaced images to match v3.12.x of GeoNetwork

alexarobu committed:
Updated image to repo

Replaced image to fix a mistake from my previous commit

alexarobu committed:
Updated ID in "CSW test" section

To match the ID of the record in Figure 9.4.2

alexarobu committed:
Updated user images to match v3.12.x

Changed the images in the repo to match the views in v3.12.x of GeoNetwork

alexarobu committed:
Update "Creating metadata from a template" section

Updated wording to reflect the extra step needed in order to access the Contribute board in v3.12

alexarobu committed:
Updated buttons to match v3.12

Replaced the button images to match the new version of GeoNetwork

alexarobu committed:
Replaced buttons to match v3.12

Updated the buttons to match how the actual buttons look in v3.12

alexarobu committed:
Updated wording for v3.12

Updated the wording around the use of |button_editor_board| in v3.12

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -57,17 +57,19 @@ Basic templates are available for users to create metadata directly on `spatiald
+	**1|** Click the |button_contribute| button in the header menu. 
 	
-	**2|** Click the |button_contribute_addrecord| button.
+	**2|** Click the |button_editor_board| button. This will enable the **Contribute** board, where all records the user has access to will be listed.
```
I think when you select the Contribute tab, the actual board is called the Editor board. However changing this may mean changing it elsewhere

alexarobu replied:
done

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -57,17 +57,19 @@ Basic templates are available for users to create metadata directly on `spatiald
+	**5|** Click on the template to be used.
 
-	**5|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the editing board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
+	**6|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the editing board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
```
I know the old text said "editing board" but I think "editor" makes more sense here?

alexarobu replied:
done

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -57,17 +57,19 @@ Basic templates are available for users to create metadata directly on `spatiald
-	**5|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the editing board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
+	**6|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the editing board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
 	
 	*Note: if the user is an editor for multiple groups or an administrator to the site, they may be presented with multiple templates of the same name. Additionally, they will be required to select which group folder to create the metadata record in.*
```
which group to assign the metadata to

alexarobu replied:
done

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -83,11 +85,13 @@ of the metadata elements are the same for all records (e.g. points of contact, e
 
 To create a metadata record by duplicating and existing record:
```
duplicating **an** existing record

alexarobu replied:
done

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -83,11 +85,13 @@ of the metadata elements are the same for all records (e.g. points of contact, e
-	**1|** Click the |button_contribute| button in the header menu. This will enable the **Contribute** board, where all records the user has access to will be listed.
+	**1|** Click the |button_contribute| button in the header menu. 
+	
+	**2|** Click the |button_editor_board| button. This will enable the **Contribute** board, where all records the user has access to will be listed.
```
See comment above about Contribute board vs editor board

alexarobu replied:
done

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -83,11 +85,13 @@ of the metadata elements are the same for all records (e.g. points of contact, e
 	
-	**3|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the **editing** board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
+	**4|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the **editing** board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
 
```
See comment above about editing board vs editor

alexarobu replied:
done

archaeogeek commented on docs/UserDoc_Chap5_Create.rst:
```diff
@@ -83,11 +85,13 @@ of the metadata elements are the same for all records (e.g. points of contact, e
-	**3|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the **editing** board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
+	**4|** Click the |button_contribute_create| button to create the new record. This will automatically take the user to the **editing** board where elements of the new record can be modified. See `Editing a metadata record <UserDoc_Chap6_Edit.html#editing-metadata>`__ for guidance on documenting the resource.
 
 	*Note: if the user is an editor for multiple groups or an administrator to the site, they will be required to select which group folder to create the duplicate metadata record in.*
```
which group to assign the metadata to

alexarobu replied:
done

archaeogeek reviewed (changes requested):
This is really good. All I've done is add a couple of fairly minor comments to some of the files. For the images, there are some where I think we might need to add more groups and logos etc so that it more closely matches the current site (rather than just showing the sample groups and logos etc). Also, I *think* scotgov has an off-white background, but I'm not sure. It might be worth double-checking the UI settings just to make sure. Hopefully, if you click "view changes" above you'll be able to see which files I viewed and which I did not. The ones I didn't view are the ones where I think we possibly need to make some changes.

alexarobu committed:
adressed pull request wording comments

alexarobu committed:
minor wording changes

alexarobu commented:
I have addressed all the minor text changes you suggested as well as a couple of similar ones I spotted in a different file to the one you made your comments on. I checked ScotGov's settings and their background colour is set to #FFFFFF. I am not sure clicking on "View Changes" worked, all files appeared as unviewed to me.

archaeogeek commented on docs/UserDoc_Chap6_Edit.rst:
```diff
@@ -92,15 +92,17 @@ To assign or change the licence category via the contribute board:
 	**1|** Click on |button_contribute| in the header.
 	
-	**2|** In the record list, select the record(s) for which the licence category will assigned or amended.
+	**2|** Click the |button_editor_board| button. This will enable the Contribute board, where all records the user has access to will be listed.
```
Editor board?

archaeogeek reviewed (changes requested):
One minor change but otherwise great!

alexarobu committed:
minor wording change

alexarobu commented:
Should be good now!

archaeogeek reviewed (commented):
Great, that's all the text changes done :-) I'll wait for the image changes before merging

alexarobu committed:
slight wording change for v3.12.x

alexarobu committed:
replaced some of the old images for v3.12.x

alexarobu commented:
I **think** these should be all the image changes done 🤞

alexarobu committed:
updated guidance around Standards

Rephrased the guidance in the Metadata & Templates subsection to reflect changes in 3.12.x
Removed section 7 on Standards as it is included in the Metadata & Templates subsection
Re-numbered the figures from what is now section 7 onwards (to reflect the removal of the Standards section)

alexarobu commented:
The commit above relates to issue #43.

alexarobu committed:
update Directories section

alexarobu committed:
update Directories figure for Gemini 2.3 changes

alexarobu commented:
The commit above relates to issue #42.

archaeogeek reviewed (approved):
Thanks, they all look great

archaeogeek commented:
I think these are all done, but I want to check whether readthedocs is parsing the 3.12 branch or not before I commit- I don't want it to render 3.12 in place of 3.10 right now :-)

archaeogeek commented:
Deprecated in favour of 4.2
